src/styles/index.scss

```diff
@@ -1,3 +1,65 @@
+$days-of-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6
+);
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+
 }
+
+.calendar {
+  width: 707px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  &__day {
```

Comment:
The BEM element `calendar__day` should not set both width and height with fixed values to maintain aspect ratio and responsiveness. Consider using percentages or a padding-top hack for maintaining aspect ratio.